{{cookiecutter.project_slug}}/.neuro/live.yml

```diff
@@ -0,0 +1,158 @@
+kind: live
+title: {{ cookiecutter.project_name }}
+id: PROJECT_SLUG_WITH_UNDERSCORES_PLACEHOLDER
```

Comment:
note: neuro-flow doesn't support slug with `-`, only with `_`. So even if the directory (slug) is named with `-`, the dir on sotrage will be named with `_`. We should fix it.

Reply:
removed at all